pilot/pkg/bootstrap/monitoring.go

```diff
@@ -44,12 +44,15 @@ var (
 	uptime = monitoring.NewDerivedGauge( // nolint: deadcode, varcheck
 		"istiod_uptime_seconds",
 		"Current istiod server uptime in seconds",
-		func() float64 {
-			return time.Since(serverStart).Seconds()
-		},
 	)
 )
 
+func init() {
+	uptime.ValueFrom(func() float64 {
```

Comment:
nit: should we make ValueFrom return the guage so you can do `uptime = monitoring.NewDervicedGauge().ValueFrom()`